filters.go

```diff
@@ -22,11 +22,15 @@ type PredicateUpdateInput struct {
 	Value string            `json:"value,omitempty"`
 }
 
+type FilterId struct {
```

Comment:
I made this type so that custom actions trigger definitions can return a more simple set of data for the filter.  This should hopefully reduce the data returned and then its up to the end user to turn that into a full object using `client.GetFilter`